specs/waku/waku-1.md

```diff
@@ -170,9 +170,13 @@ packet-format = "[" packet-code packet-format "]"
 
 required-packet = 0 status /
                   1 messages /
-		  22 status-update /
+                  22 status-update /
 
-optional-packet = 126 p2p-request / 127 p2p-message
+optional-packet = 11 batch-ack /
```

Comment:
The batch-ack and message-response content is explained in ABNF located in the confirmations section. Perhaps that should be moved here?